src/hazelcore/semantics/Action.re

```diff
@@ -3273,12 +3363,17 @@ module Exp = {
       Succeeded(SynDone((new_ze, ty, u_gen)));
     | (Construct(SAsc), CursorE(_)) => Failed
 
+    /* It is a temporary fix so that a new hole isn't created when a dot is inputted by itself.
+       It will later be replaced by invalid var and handled by make syntax */
```

Comment:
Temporary fix so that a new hole isn't created when a dot is inputted by itself (cf mk_[syn|ana]_text)
TODO remove once we have InvalidVar holes

Reply:
Think you'll also need to add this case to `Pat.[syn|ana]_perform_operand`